benchmark/realnest/04_list_transform_plus_list_aggregate.benchmark

```diff
@@ -0,0 +1,62 @@
+# name: benchmark/realnest/04_list_transform_plus_list_aggregate.benchmark
+# description: select average from transformed list and group by, having
```

Comment:
Calculate different list averages after transforming the list. The `AVG` serves to limit the result set size and has no significant impact on performance.

We should be able to remove the `GROUP BY`.
```sql
D select avg(len(j)) from l;
┌─────────────┐
│ avg(len(j)) │
│   double    │
├─────────────┤
│         2.0 │
└─────────────┘
D from l;
┌──────────┐
│    j     │
│ int32[]  │
├──────────┤
│ [42, 43] │
└──────────┘
```